Lets Do This/Models/DSOUser.m

```diff
@@ -33,23 +33,22 @@ - (instancetype)initWithDict:(NSDictionary *)dict {
     self = [super init];
 
     if (self) {
-        self.userID = dict[@"_id"];
+        _userID = dict[@"_id"];
         // Hack to hotfix inconsistent API id property: https://github.com/DoSomething/LetsDoThis-iOS/issues/340
-        if (!self.userID) {
-            self.userID = [dict valueForKeyAsString:@"id" nullValue:@"null-id"];
+        if (!_userID) {
+            _userID = [dict valueForKeyAsString:@"id" nullValue:@"null-id"];
         }
         if ([dict objectForKey:@"country"]) {
-            self.countryCode = dict[@"country"];
+            _countryCode = dict[@"country"];
         }
-        self.firstName = [dict valueForKeyAsString:@"first_name" nullValue:@"Doer"];
-        self.email = dict[@"email"];
-        self.sessionToken = dict[@"session_token"];
-        self.mutableCampaignSignups = [[NSMutableArray alloc] init];
+        _firstName = [dict valueForKeyAsString:@"first_name" nullValue:@"Doer"];
+        _email = dict[@"email"];
+        _sessionToken = dict[@"session_token"];
+        _mutableCampaignSignups = [[NSMutableArray alloc] init];
 		
         if (dict[@"photo"]) {
-            self.photo = nil;
```

Comment:
@tongxiang You ref this line in #647 -- which I've removed since you opened the PR -- I'm not sure why we need it in the first place (or who added it -- def could have been me)